src/connect.js

```diff
@@ -4,7 +4,9 @@ import { core, object } from 'metal';
 import JSXComponent from 'metal-jsx';
 
 const defaultMapStateToProps = () => ({});
-const defaultMapDispatchToProps = dispatch => ({ dispatch });
+const defaultMapDispatchToProps = dispatch => ({
+	dispatch
+});
 const defaultMergeProps = (stateProps, dispatchProps, parentProps) => {
 	return object.mixin({}, stateProps, dispatchProps, parentProps);
```

Comment:
This commit made me realize that the order of mixin here is wrong. It's supposed to be `parentProps, stateProps, dispatchProps`, [like in react-redux](https://github.com/reactjs/react-redux/blob/4.x/src/components/connect.js#L13), which makes sense since the new props should have a preference.

This was only working because I was passing them in the wrong order inside `getChildProps_`, but now that you've fixed that, this needs to be fixed too. Can you update this?